src/app/_mermaid_common/libs/services/connectivity.svc.js

```diff
@@ -32,11 +33,15 @@ angular.module('mermaid.libs').service('connectivity', [
       }
     };
 
-    var toggleDisabled = function() {
+    const toggleDisabled = function() {
+      if (pingState === false) {
```

Comment:
If this is the right logic, then all these line could be replaced with 
`return pingState !== true` right?
Is that enough for the fix?